config/crd/bases/secrets.doppler.com_dopplersecrets.yaml

```diff
@@ -36,6 +36,9 @@ spec:
           spec:
             description: DopplerSecretSpec defines the desired state of DopplerSecret
             properties:
+              config:
```

Comment:
Should we document somewhere that these only need to be specified when using a non-service token?